From e1b0ef7df2624fbfca4a02811dcd929e54babac4 Mon Sep 17 00:00:00 2001
From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Tue, 14 Aug 2018 12:09:45 +0300
Subject: [PATCH 0173/2940] drm/amd/display: indent an if statement

The if statement isn't indented and it makes static checkers complain.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_resource.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
index 2fe89674fc2e..8f1be9cff77c 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
@@ -367,7 +367,7 @@ static bool is_dp_and_hdmi_sharable(
 
 	if (stream1->clamping.c_depth != COLOR_DEPTH_888 ||
 	    stream2->clamping.c_depth != COLOR_DEPTH_888)
-	return false;
+		return false;
 
 	return true;
 
-- 
2.17.1

